src/asset-swapper/utils/market_operation_utils/index.ts

```diff
@@ -720,12 +740,27 @@ export class MarketOperationUtils {
             } else {
                 // A firm quote is being requested, and firm quotes price-aware enabled.
                 // Ensure that `intentOnFilling` is enabled and make the request.
-                const firmQuotes =
+
+                const [v1Quotes, v2Quotes] =
                     rfqt.rfqClient === undefined
-                        ? []
-                        : (
-                              await rfqt.rfqClient.getV1QuotesAsync({
-                                  altRfqAssetOfferings: filteredOfferings,
+                        ? [[], []]
+                        : await Promise.all([
+                              rfqt.rfqClient
+                                  .getV1QuotesAsync({
+                                      altRfqAssetOfferings: filteredOfferings,
+                                      assetFillAmount: amount,
+                                      chainId: this._sampler.chainId,
+                                      comparisonPrice: wholeOrderPrice,
+                                      integratorId: rfqt.integrator.integratorId,
+                                      intentOnFilling: rfqt.intentOnFilling,
+                                      makerToken,
+                                      marketOperation: side,
+                                      takerAddress: rfqt.takerAddress,
+                                      takerToken,
+                                      txOrigin: rfqt.txOrigin,
+                                  })
+                                  .then((res) => res.quotes),
+                              rfqt.rfqClient.getV2QuotesAsync({
                                   assetFillAmount: amount,
                                   chainId: this._sampler.chainId,
                                   comparisonPrice: wholeOrderPrice,
```

Comment:
Do we need `comparisonPrice` as a request parameter? I don't think we have it in rfqt v2 handler...

Reply:
No, will remove